BRAD/justchat.py

```
@@ -0,0 +1,130 @@
+from langchain.chains import ConversationChain
+from langchain.prompts import PromptTemplate
+from langchain_core.prompts.prompt import PromptTemplate
+from langchain.docstore.document import Document
+from langchain_chroma import Chroma
+from langchain_community.document_loaders import PyPDFLoader
+from langchain_community.document_loaders import TextLoader
+from langchain_community.embeddings.sentence_transformer import (
+    SentenceTransformerEmbeddings,
+)
+from sklearn.metrics.pairwise import cosine_similarity
+from langchain_text_splitters import CharacterTextSplitter
```

Ruff

Redefinition of unused `CharacterTextSplitter` from line 20

(F811)